### ISSUE TITLE
Glitches with in_card and Home Assistant 2022.11

### ISSUE THREAD

**Issue by McGiverGim** [jcwillox/lovelace-canary#21]
### The problem

With latest Home Assistant, there are glitches where the cards join. Here a capture of three cards in one:

![image](https://user-images.githubusercontent.com/2673520/200112553-343b59bc-cda5-48b8-bd54-73499dcc5983.png)

### What version of Canary has the issue?

0.3.4

### What version of Home Assistant are you running?

Home Assistant 2022.11.1

### What version of the Frontend are you running?

Frontend 20221102.1

### Example YAML snippet

```yml
Any with `in_card: true`
```

### Anything in the logs that might be useful for us?

```python3
Nothing
```

### Additional information

None

**Comment by bedson87** [comment]
yup - https://github.com/custom-cards/stack-in-card/pull/47

**Comment by birdwing** [comment]
Issue also appears with the no_card option as well.

**Comment by iMiMx** [comment]
So it looks like we probably need a 'border: none' in:

https://github.com/jcwillox/lovelace-canary/blob/main/src/const.ts

**Comment by iMiMx** [comment]
As a workaround, you can use the following in Lovelace:

```
canary_style:
  border: none;
```

Full example, where I had a horizontal line going through the 'boost' buttons:

```
cards:
  - entity: climate.hallway
    name: Hallway
    control: false
    step_layout: row
    type: custom:simple-thermostat
  - buttons:
      - entity: script.hallway_heating_boost_1hour
        icon: mdi:clock-time-one
        name: +1 Hour
      - entity: script.hallway_heating_boost_2hour
        icon: mdi:clock-time-two
        name: +2 Hour
      - entity: script.hallway_heating_boost_cancel
        icon: mdi:close-circle
        name: Reset / Cancel
    type: custom:paper-buttons-row
  - entities:
      - style:
          height: 20px
          margin-left: auto
          margin-right: auto
          width: 80%
        type: divider
    type: entities
    canary_style:
      border: none;
type: vertical-stack
in_card: true
```

**Comment by jcwillox** [comment]
Yeah, this is definitely an issue, I'll try to get it fixed soon 👍

**Comment by McGiverGim** [comment]
Some news about that @jcwillox ?

**Comment by jcwillox** [comment]
Yeah very sorry that it's taken so long, I have been very busy with the job I just started + other things, I was planning to sort it out this weekend but there was a breaking change with HA 2022.12 for my other plugin [paper-buttons-row](https://github.com/jcwillox/lovelace-paper-buttons-row), which took priority as there was no simple fix for it. That's done now so hopefully the canary fix will be out tomorrow night.

It also took a little longer as there hasn't been a release since I migrated the codebase to typescript so I need to make sure I didn't break anything back when I did that.

(Honestly no idea where the last month went 👀)

Just a reminder to anyone that stumbled upon this, that this will temporarily fix the issue
```yaml
canary_style:
  border: none
```
